src/Controls/samples/Controls.Sample/Pages/XamlPage.xaml

```diff
@@ -1,13 +1,30 @@
 ﻿<controls:BasePage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
                    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                    xmlns:controls="clr-namespace:Maui.Controls.Sample.Controls"
+                   xmlns:layout="clr-namespace:Microsoft.Maui.Controls.Layout2;assembly=Microsoft.Maui.Controls"
                    x:Class="Maui.Controls.Sample.Pages.XamlPage">
 
-    <StackLayout>
-        <Label 
-               Text="Welcome to Microsoft MAUI!"
-               VerticalOptions="CenterAndExpand"
-               HorizontalOptions="CenterAndExpand" />
-    </StackLayout>
+    <layout:GridLayout x:Name="MyLayout" Margin="40" BackgroundColor="AntiqueWhite" HorizontalOptions="Start"
+                       RowSpacing="5" ColumnSpacing="2" layout:RowDefinitions="100,200" layout:ColumnDefinitions="Auto,Auto">
```

Comment:
Please tell me the defaults for these things (spacings, margins, paddings) are all 0? All my life I have been adding default styles to my apps to reset. I feel like I am on the web with reset CSS...

I am not going to look myself, I am too scared.

Reply:
Too late, I scrolled and saw a sweet, sweet 0d! ❤️